packages/database/src/databases/client.ts

```diff
@@ -25,9 +26,9 @@ import { MongoDatabase } from "../base/index.js";
 
 const CLIENT_TABLES = [
 	{
-		collectionName: TableNames.emails,
-		modelName: "Email",
-		schema: UserDataSchema,
+		collectionName: TableNames.accounts,
+		modelName: "Account",
```

Comment:
is this not going to move all the data?

Reply:
the data is already moved. This code is no longer functional because stuff was migrated to accounts